sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/webpack-assets/build.sbt

```diff
@@ -51,10 +47,14 @@ InputKey[Unit]("html") := {
   assert(files.length == assetsCount)
   // Check all files are present
   assert(files.map(_.data.exists).forall(_ == true))
+  // There is only one app file
+  assert(files.count(_.metadata.get(BundlerFileTypeAttr) == Some(BundlerFileType.Application)) == 1)
```

Comment:
Not sure to understand how this test worked before, as from my understanding `LibraryOnly` mode produces three files: library, loader & app.